models/shopify__daily_shop.sql

```diff
@@ -0,0 +1,213 @@
+with shop as (
+
+    select *
+    from {{ var('shopify_shop') }}
+),
+
+calendar as (
+
+    select *
+    from {{ ref('shopify__calendar') }}
+    where cast({{ dbt.date_trunc('day','date_day') }} as date) = date_day
+
+),
+
+shop_calendar as (
+
+    select
+        calendar.date_day,
```

Comment:
Can we truncate this to the day. It is a bit confusing as a timestamp.

Reply:
done